### PR TITLE
2.0-preview

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -19,16 +19,33 @@
...
-    "paper-styles": "PolymerElements/paper-styles#^1.0.0",
-    "paper-button": "PolymerElements/paper-button#^1.0.0",
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "test-fixture": "PolymerElements/test-fixture#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "paper-styles": "PolymerElements/paper-styles#2.0-preview",
+    "paper-button": "PolymerElements/paper-button#2.0-preview",
+    "iron-component-page": "PolymerElements/iron-component-page#2.0-preview",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
```
Comment (bicknellr) [inline bower.json]: WCT 4 & 6 have the right versions of test-fixture now.

**Comment by bicknellr** [comment]
The demo is broken in 2.x because of [a change to IronFormElementBehavior](https://github.com/PolymerElements/iron-form-element-behavior/pull/30#issuecomment-293425584).

**Comment by bicknellr** [comment]
(For reference, I'm now the owner of this PR.)

**Comment by bicknellr** [comment]
Restarting the tests in case deps have changed.

File: bower.json (hunk 1 above)
Comment (notwaldorf) [reply bower.json]: This is your element now and i agree with this comment :)

**Review by notwaldorf — APPROVED** [review]
LGTM, provided that bower comment you left

File: bower.json (hunk 1 above)
Comment (bicknellr) [reply bower.json]: fixed

**Comment by bicknellr** [comment]
(restarting tests which failed due to a network timeout)

**Pull request merged**